bin/autodetect/Main.cc

```diff
@@ -203,6 +188,22 @@ int main(int argc, char** argv) {
         return EXIT_FAILURE;
     }
 
+    std::string anomalyJobConfigJson;
+    bool couldReadConfigFile;
+    std::tie(anomalyJobConfigJson, couldReadConfigFile) =
+        ml::core::CStringUtils::readFileToString(configFile);
+    if (couldReadConfigFile == false) {
+        LOG_FATAL(<< "Failed to read config file '" << configFile << "'");
+        return EXIT_FAILURE;
+    }
+    // For now we need to reference the rule filters parsed by the old-style
+    // field config.
+    ml::api::CAnomalyJobConfig jobConfig(fieldConfig.ruleFilters());
```

Comment:
nit: prefer brace initialization

```suggestion
    ml::api::CAnomalyJobConfig jobConfig{fieldConfig.ruleFilters()};
```